qiskit/algorithms/quantum_time_evolution/variational/var_qte.py

```diff
@@ -160,15 +160,17 @@ def evolve_helper(
         # TODO bind Hamiltonian?
 
         self._variational_principle._lazy_init(
-            hamiltonian, initial_state, init_state_param_dict, self._regularization
+            hamiltonian, initial_state, init_state_parameters,
+            # self._regularization
```

Comment:
Please remove.

Reply:
We don't need the regularization in the lazy init function.
Pls check my changes.

Reply:
I just meant that the commented fragment should be deleted. I understand it is not needed and therefore was commented out.